<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EstatusClaveDialogBase</class>
 <widget class="QDialog" name="EstatusClaveDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>166</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>315</width>
    <height>165</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>316</width>
    <height>166</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Claves</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background : rgb(250,238,224);
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget{
background : rgb(255,255,255);
}</string>
     </property>
     <widget class="QPushButton" name="botonConfirmar">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background : rgb(174, 116, 0);
color : rgb(255, 255, 255);
border-radius : 4px;
}
QPushButton::hover{
background : rgb(104, 69, 13);
color : rgb(255, 255, 255);
border-radius : 4px;
}
QPushButton:disabled {
background: rgb(217, 217, 217)
}</string>
      </property>
      <property name="text">
       <string>Confirmar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="botonCancelar">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background : rgb(174, 116, 0);
color : rgb(255, 255, 255);
border-radius : 4px;
}
QPushButton::hover{
background : rgb(104, 69, 13);
color : rgb(255, 255, 255);
border-radius : 4px;
}
QPushButton:disabled {
background: rgb(217, 217, 217)
}</string>
      </property>
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>271</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Nuevo estatus de la clave anterior</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelTituloEditar">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Clave anterior:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leClaveAnterior">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>001</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelTituloEditar_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Estatus:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboManzana">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox QAbstractItemView::item { min-height: 25; font-family: &quot;Century Gothic&quot;;}
QListView{color: black ; font-size: 16px; outline:none; font-weight:bold; font-family: &quot;Century Gothic&quot;;}

QComboBox{
background : rgb(250,238,224);
}
QComboBox:item:selected {
color: black ;
}</string>
      </property>
      <item>
       <property name="text">
        <string>INACTIVA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ACTIVA</string>
       </property>
      </item>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
